docs/codeql/codeql-workshops/java/apache-struts-online.txt

```diff
@@ -0,0 +1,2 @@
+There is also the full writeup at
```

Comment:
Can we remove the `Introduction to CodeQL - Java.pdf` file in this folder? I think it's generated from other contents in this repo.